rs/sns/governance/src/governance/tests/fail_stuck_upgrade_in_progress_tests.rs

```diff
@@ -114,7 +114,8 @@ fn test_does_nothing_if_there_is_no_upgrade_in_progress() {
         Box::new(DoNothingLedger {}),
         Box::new(DoNothingLedger {}),
         Box::new(FakeCmc::new()),
-    );
+    )
+    .enable_test_features();
```

Comment:
Why is this needed in these tests?  Seems a bit surprising.

Reply:
I removed it!